src/components/WebexLocalMedia/WebexLocalMedia.js

```diff
@@ -3,24 +3,29 @@ import PropTypes from 'prop-types';
 import {Spinner} from '@momentum-ui/react';
 
 import {WebexAvatar} from '../';
-import {useMe, useMeeting, useStream} from '../hooks';
+import {useElementDimensions, useMe, useMeeting, useStream} from '../hooks';
 
 import './WebexLocalMedia.scss';
 
 /**
- * Webex Local Media component displays the user's local video
+ * Webex Local Media component displays the user's local video.
  *
- * @param {object} props
- * @returns {object} JSX of the component
+ * @param {string} props.meetingID  ID of the meeting from which to obtain local media
+ * @returns {Object} JSX of the component
  */
 export default function WebexLocalMedia({meetingID}) {
+  const [mediaRef, {width}] = useElementDimensions();
   const {localVideo} = useMeeting(meetingID);
   const {ID} = useMe();
   const videoRef = useStream(localVideo);
+  const noMediaClass = localVideo ? '' : ' no-media';
+  const sizeClass = width >= 425 ? ' local-media-large' : ''; // Standard large phone size
   const disabledVideo = ID ? <WebexAvatar personID={ID} displayStatus={false} /> : <Spinner />;
 
   return (
-    <div className="local-media">{localVideo ? <video ref={videoRef} playsInline autoPlay /> : disabledVideo}</div>
+    <div ref={mediaRef} className={`local-media${sizeClass}${noMediaClass}`}>
```

Comment:
Not needed, but if you want to reduce the complexity of this, you can use the tool we use in the widgets called "classnames" and allows you to throw classnames into an array like:

```js
const localMediaClasses = ['local-media'];
if (width >= 425) localMediaClasses.push(' local-media-large');
if (!localVideo) localMediaClasses.push('no-media');
<div className={classNames(localMediaClasses)}>
```

https://www.npmjs.com/package/classnames

Reply:
Yeah, I'm aware of it. I remember when trying to dive into the widget code the first few times, it actually made it harder for me to figure out what the code was doing. I had to spend extra time figuring out and googling what this `classNames` function-looking thing was supposed to be doing (was that React syntax? why isn't that in the docs? is that a third-party?), and it made cognitive harder for to to me to keep track of the CSS class names.
I would personally avoid it

Reply:
Not to be argumentative, but knowing that all the class names in an array are going to be spread properly seems like MUCH less cognitive overhead than having to know which variables need spaces before them or nulled out when not needed. Like I said above though, not needed and not blocking the PR.

Reply:
I used the utility after all. It was already part of our dev dependencies 😛 and I found their object syntax much more readable